drivers/adc/Kconfig.mcux

```diff
@@ -53,6 +53,28 @@ config ADC_MCUX_ADC16_VREF_ALTERNATE
 
 endchoice
 
+config ADC16_ENABLE_EDMA
+	bool "ENABLE EDMA for adc driver"
+	depends on HAS_MCUX_ADC16 && HAS_MCUX_EDMA
+	help
+	  Enable the MCUX ADC16 driver.
+
+if ADC16_ENABLE_EDMA
+
+config ADC16_INIT_PRIORITY
+	int "Init priority"
+	default 70
+	help
+	  Device driver initialization priority.
+
+config ADC_HW_TRIGGER
+	bool "ADC HW TRIGGER"
+	default y
```

Comment:
Should this default to `n` to keep existing behaviour?

Reply:
ok

Reply:
ok